project/cmake/scripts/windows/Macros.cmake

```diff
@@ -14,9 +15,16 @@ function(core_link_library lib wraplib)
     endforeach()
 
     add_library(wrap_${lib} SHARED ${SOURCES_ABS})
-    set_target_properties(wrap_${lib} PROPERTIES OUTPUT_NAME ${wraplib})
+    set_target_properties(wrap_${lib} PROPERTIES OUTPUT_NAME lib${lib})
     target_include_directories(wrap_${lib} PRIVATE ${INCLUDES})
     target_compile_definitions(wrap_${lib} PRIVATE ${DEFINITIONS})
+    
+    if (MSVC)
+      string(REPLACE "/lib${lib}" "" libdir ${wraplib})
+      set_target_properties(wrap_${lib} PROPERTIES  RUNTIME_OUTPUT_DIRECTORY_RELEASE "${CMAKE_BINARY_DIR}/${libdir}"
```

Comment:
Only one space between <code>PROPERTIES</code> and <code>RUNTIME...</code> please